src/coreclr/jit/llvmlower.cpp

```diff
@@ -929,12 +916,15 @@ void Llvm::lowerVirtualStubCallAfterArgs(
     stubAddr->gtFlags |= GTF_IND_NONFAULTING;
     CurrentRange().InsertBefore(callNode, stubAddr);
 
+    GenTree* thisForStub  = _compiler->gtNewLclvNode(thisArgLclNum, TYP_REF);
```

Comment:
Nit: I would prefer to keep the original order for how `thisForStub` is created/inserted. It is meant to match the final operand layout (SS -> `this` -> stub call target as `stubAddr`).

Reply:
Right, thanks, I've put that back now.